libutils/sequence.c

```diff
@@ -553,6 +553,7 @@ Seq *SeqStringDeserialize(const char *const serialized)
             || NULL == (new_str = ValidDuplicate(src, length))
             || src[length] != '\n')
         {
+            free(new_str);
```

Comment:
This can result in an invalid free if any of the first three if's hit.

Reply:
@karlhto no, look further up at the diff. I initialized it to NULL. `free(NULL)` is valid.

Reply:
From Linux man page:

> If ptr is NULL, no operation is performed.

From C99 standard:

> If ptr is a null pointer, no action occurs.

Reply:
Alright :+1: